ui/app/AppLayouts/Onboarding2/UseRecoveryPhraseFlow.qml

```diff
@@ -17,6 +17,7 @@ OnboardingStackView {
 
     required property var passwordStrengthScoreFunction
     required property var isSeedPhraseValid
+    required property var isSeedPhraseValidForImport
```

Comment:
I think for `UseRecoveryPhraseFlow` just `isSeedPhraseValid` is sufficient, the distinction can be done on the upper level

Reply:
Hmm not sure, we have to do this only for some of the flow types here, and also access the seedphrase panel to set the error from outside; would be really cumbersome from 2 levels above

Reply:
I mean, in general, it could be done (later) together with some async seedphrase validation. The idea is that we'd return a result signal there with a tuple of `bool result; string error` and use that in the seedphrase panel to validate and display any error message.

Reply:
But this is well beyond the scope of this fix PR; would require some substantial refactoring of the (shared) underlying `EnterSeedPhrase` panel